services/meta/client.go

```diff
@@ -983,6 +983,12 @@ func snapshot(path string, data *Data) error {
 		return err
 	}
 
+	if _, err := os.Stat(file); err == nil {
```

Comment:
Can you just check `runtime.GOOS`? I don't think we need to go as far as requiring conditional compilation.